godot-google-play-billing/src/main/java/org/godotengine/godot/plugin/googleplaybilling/GodotGooglePlayBilling.java

```diff
@@ -216,14 +213,14 @@ public Dictionary confirmPriceChange(String sku) {
 	@UsedByGodot
 	public Dictionary purchase(String sku) {
 		return purchaseInternal("", sku, 
-			BillingFlowParams.ProrationMode.UNKNOWN_SUBSCRIPTION_UPGRADE_DOWNGRADE_POLICY);
+			BillingFlowParams.SubscriptionUpdateParams.ReplacementMode.UNKNOWN_REPLACEMENT_MODE);
```

Comment:
`ProrationMode` is [deprecated](https://developer.android.com/reference/com/android/billingclient/api/BillingFlowParams.ProrationMode): `use SubscriptionUpdateParams.ReplacementMode instead`